### PR TITLE
Scaffold front page

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: front-page.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?php
+get_header();
+
+$front_page_projects = IGV_get_option('_igv_front_page');
+$front_page_projects = preg_split('/[\s,]+/', $front_page_projects);
```
Comment (mixcas) [inline front-page.php:6]: could u explain this on comments, @varffs?

Comment (varffs) [inline front-page.php:6]: yh sure, its just a regex that I found the splits both commas and blank spaces

File: lib/theme-options.php [hunk 2]
```diff
@@ -101,6 +101,25 @@ function add_options_page_metabox() {
...
+
+    $options_metabox->add_field( array(
+      'name' => __( 'Front Page options', 'cmb2' ),
+      'desc' => __( '', 'cmb2' ),
+      'id'   => $this->prefix . 'front_title',
+      'type' => 'title',
+    ) );
+
+    $options_metabox->add_field( array(
+      'name'        => __( 'Related post' ),
```
Comment (mixcas) [inline lib/theme-options.php]: are these "related posts" ?

File: front-page.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+// split comma delimited string into array
+$front_page_projects = preg_split('/[\s,]+/', $front_page_projects);
+// get random index for array
+$random_index = rand(0, (count($front_page_projects) - 1));
+
+$front_page_gallery = get_post_meta($front_page_projects[$random_index], '_igv_gallery');
+?>
+  </div>
+
+  <a href="<?php echo get_the_permalink($front_page_projects[$random_index]); ?>">
```
Comment (mixcas) [inline front-page.php:14]: I'm not sure about this link… i mean, the whole screen is clickable. smells like bad UX

Comment (varffs) [inline front-page.php:14]: Except the menu so its functionally a splash which is the design. I think its fine really

File: lib/theme-options.php (hunk 2 above)
Comment (varffs) [inline lib/theme-options.php]: Oh nope

File: front-page.php (hunk 3 above)
Comment (mixcas) [inline front-page.php:14]: IT SUX

**Pull request merged**